cmd/renterd/main.go

```diff
@@ -115,6 +115,8 @@ func main() {
 		apiPassword string
 		node.BusConfig
 	}
+	busCfg.PersistInterval = 10 * time.Minute
```

Comment:
if it's on the `busCfg` shouldn't we make it a command line flag while we're at it?

Reply:
I would not default to making everything in the config files a command line flag. Because it just bloats the `-h` flag without being a very useful value to tweak.